### DISCUSSION TITLE
Can't log in to Spotify using cookies

### DISCUSSION THREAD

**Discussion by AxeForward** [KRTirtho/spotube#986] [Q&A]
I am a user from mainland China, and I can log in to Spotify only by using V2Ray proxy. I can log in normally on the web page, but after following the prompts and pasting sp_dc and sp_key into the corresponding locations, a login error occurs.
![image](https://github.com/KRTirtho/spotube/assets/95564356/197e2e2e-9d54-4b9f-a8ea-13787b63d7ef)
My operating system is Windows 11 Home Edition 22H2. Spotube version is 3.4.0.

**Comment by Ecustkiller** [comment]
我也是这个问题

**Comment by oldchow** [comment]
![捕获](https://github.com/KRTirtho/spotube/assets/87064579/d20d77f9-c334-495d-a828-7073c8303ac8)
same here

**Comment by KRTirtho** [comment]
This issue only seems to happen in Mainland China. I guess spotify uses a different endpoint there for login.
Can you share the Spotify url you use?

**Reply by oldchow** [reply]
Do you mean the url I use for logging in? it's https://open.spotify.com/?  , which spotify always directs into when i go to spotify.com.

**Reply by KRTirtho** [reply]
so you can't use Spotify web version?

**Reply by oldchow** [reply]
yes, i can, it will look like this
![捕获](https://github.com/KRTirtho/spotube/assets/87064579/cea50fe1-b99c-487f-a4f5-ef5187eb0c06)

**Reply by KRTirtho** [reply]
Then, you're having the same issue as #974

**Reply by KRTirtho** [reply]
Happens when you use Proxy
